### PR TITLE
Revert merge clean up

### PR DESCRIPTION
**Pull request by maxlath**
- also revert side-effects that weren't redirected yet: claims transfer, claims redirections
- added tests

### CODE REVIEW

File: api_tests/entities/merge.test.coffee [hunk 1]
```diff
@@ -74,3 +74,40 @@ describe 'entities:merge', ->
     .catch undesiredErr(done)
 
     return
+
+  it 'should transfer claims', (done)->
```
Comment (jum-s) [inline api_tests/entities/merge.test.coffee]: suggestion : `should transfer work claims to another work`

File: api_tests/entities/revert_merge.test.coffee [hunk 2]
```diff
@@ -31,3 +30,102 @@ describe 'entities:revert-merge', ->
...
+      .then (res)->
+        res.entities[workB.uri].labels.zh.should.equal labelA
+        adminReq 'put', '/api/entities?action=revert-merge', { from: workA.uri }
+      .then -> nonAuthReq 'get', "/api/entities?action=by-uris&uris=#{workB.uri}"
+      .then (res)->
+        should(res.entities[workB.uri].labels.zh).not.be.ok()
+        done()
+    .catch undesiredErr(done)
+
+    return
```
[the remark this answers is not in the document]
Comment (maxlath) [reply api_tests/entities/revert_merge.test.coffee]: addressed by 98ca1e2

File: api_tests/entities/merge.test.coffee (hunk 1 above)
Comment (maxlath) [reply api_tests/entities/merge.test.coffee]: it's testing for entities in general: `should transfer entity claims to the target entity`?

File: api_tests/entities/merge.test.coffee [hunk 3]
```diff
@@ -130,3 +130,27 @@ describe 'entities:merge', ->
...
+      .then ->
+        adminReq 'put', '/api/entities?action=merge',
+          from: humanA.uri
+          to: humanB.uri
+      .then -> nonAuthReq 'get', "/api/entities?action=by-uris&uris=#{work.uri}"
+      .then (res)->
+        res.entities[work.uri].claims['wdt:P50'][0].should.equal humanB.uri
+      .then -> nonAuthReq 'get', "/api/entities?action=history&id=#{work._id}"
+      .then (res)->
+        res.patches[2].context.redirectClaims
```
[the remark this answers is not in the document]
Comment (maxlath) [reply api_tests/entities/merge.test.coffee]: addressed by 0dc7f50

**Pull request merged**